var/spack/repos/builtin/packages/py-invoke/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyInvoke(PythonPackage):
+    """Pythonic task execution"""
+
+    homepage = "http://www.pyinvoke.org/"
+    url      = "https://pypi.io/packages/source/i/invoke/invoke-1.2.0.tar.gz"
+
+    version('1.2.0', 'dc492f8f17a0746e92081aec3f86ae0b4750bf41607ea2ad87e5a7b5705121b7')
+
+    extends('python')
+    depends_on('python@2.7:2.8,3.4:')
```

Comment:
`type=('build', 'run')`

Reply:
Also fixed in [073ca50](https://github.com/spack/spack/commit/073ca504514c22d79e6ae03a12084b7fa47b9bca).